roles/check-keys-password-teku/tasks/main.yaml

```diff
@@ -43,6 +43,16 @@
   with_items: '{{importing_key_cli.stdout_lines}}'
   when: validator_keys is not defined and (validator_log.stdout is search('Failed to decrypt keystore') or validator_log.stdout is search('Keystore password cannot be empty'))
 
+- name: Remove keystores files from launchpad 'import-config'
+  shell: rm "{{ e2dc_install_path }}/launchpad/*.json"
```

Comment:
you can use `file:` with `state: absent` https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-state